data/net/minecraft/client/multiplayer/chat/ChatListener.mapping

```diff
@@ -2,70 +2,70 @@ CLASS net/minecraft/client/multiplayer/chat/ChatListener
 	METHOD <init> (Lnet/minecraft/client/Minecraft;)V
 		ARG 1 minecraft
 	METHOD evaluateTrustLevel (Lnet/minecraft/network/chat/PlayerChatMessage;Lnet/minecraft/network/chat/Component;Lnet/minecraft/client/multiplayer/PlayerInfo;Ljava/time/Instant;)Lnet/minecraft/client/multiplayer/chat/ChatTrustLevel;
-		ARG 1 message
-		ARG 2 decoratedServerContent
-		ARG 3 senderPlayerInfo
-		ARG 4 timeStamp
+		ARG 1 chatMessage
+		ARG 2 message
+		ARG 3 playerInfo
+		ARG 4 timestamp
 	METHOD guessChatUUID (Lnet/minecraft/network/chat/Component;)Ljava/util/UUID;
 		ARG 1 message
 	METHOD handleChatHeader (Lnet/minecraft/network/chat/SignedMessageHeader;Lnet/minecraft/network/chat/MessageSignature;[B)V
 		ARG 1 header
-		ARG 2 headerSignature
+		ARG 2 signature
 		ARG 3 bodyDigest
 	METHOD handleChatMessage (Lnet/minecraft/network/chat/PlayerChatMessage;Lnet/minecraft/network/chat/ChatType$Bound;)V
 		COMMENT Called whenever a player chat message is received from the server.
-		ARG 1 message
+		ARG 1 chatMessage
 		ARG 2 boundChatType
 	METHOD handleMessage (Lnet/minecraft/client/multiplayer/chat/ChatListener$Message;)V
 		ARG 1 message
 	METHOD handleSystemMessage (Lnet/minecraft/network/chat/Component;Z)V
 		ARG 1 message
-		ARG 2 actionBar
+		ARG 2 isOverlay
 	METHOD isSenderLocalPlayer (Ljava/util/UUID;)Z
-		ARG 1 profileId
+		ARG 1 sender
 	METHOD logPlayerHeader (Lnet/minecraft/network/chat/SignedMessageHeader;Lnet/minecraft/network/chat/MessageSignature;[B)V
 		ARG 1 header
-		ARG 2 headerSignature
+		ARG 2 headSignature
```

Comment:
This is not consistent with the other change. headSignature vs signature. Choose one but I think headerSignature is best.

Reply:
Undone?